src/components/FarmDualCard/FarmDualCardDetails.tsx

```diff
@@ -323,157 +335,212 @@ const FarmDualCardDetails: React.FC<{
       ? '< $0.001'
       : '$' + earnedUSD.toLocaleString();
 
+  const valueOfTotalStakedAmountInUSDC =
+    valueOfTotalStakedAmountInBaseToken &&
+    USDPrice?.quote(valueOfTotalStakedAmountInBaseToken);
+
+  const tvl = valueOfTotalStakedAmountInUSDC
+    ? `$${valueOfTotalStakedAmountInUSDC.toFixed(0, { groupSeparator: ',' })}`
+    : `${valueOfTotalStakedAmountInBaseToken?.toSignificant(4, {
+        groupSeparator: ',',
+      }) ?? '-'} ETH`;
+
+  const rewards = useMemo(() => {
+    if (!stakingInfo) return 0;
+    return (
+      stakingInfo?.rateA * stakingInfo?.quickPrice +
+      stakingInfo?.rateB * Number(stakingInfo?.rewardTokenBPrice)
+    );
+  }, [stakingInfo]);
+
+  const poolRateA = `${stakingInfo?.totalRewardRateA
+    ?.toFixed(2, { groupSeparator: ',' })
+    .replace(/[.,]00$/, '')} ${stakingInfo?.rewardTokenA?.symbol}  / day`;
```

Comment:
this should be moved into a format function to be re used.
Also why are you stripping the 00's from the end are you use this won't cause issues for  with localization of the display of some numbers in other countries?

Reply:
`replace(/[.,]00$/, '')` is for stripping 00 when the value is integer. i.e when the rate is 300, `toFixed` returns `300.00` and this will strip `.00` and return `300`